Engine/Source/ModuleEditor.cpp

```diff
@@ -254,6 +265,80 @@ void ModuleEditor::ShowMainMenuBar() {
 	if (mLoadSceneOpen) {
 		OpenLoadScene();
 	}
+
+	if (quadtreeDebug->IsOpen() == false) {
+		mQuadtreePanel = false;
+	}
+	else {
+		mQuadtreePanel = true;
+	}
+
+	if (debugPanel->IsOpen() == false) {
+		mDebugPanel = false;
+	}
+	else {
+		mDebugPanel = true;
+	}
+
+	if (timerPanel->IsOpen() == false) {
```

Comment:
![image](https://github.com/Assigment2Company/Assigment2/assets/45202028/ac2f57f9-4242-4681-b06e-7f38a5340695)
We should do it this way, instead of creating booleans